### PR TITLE
[CollecticielBundle] fixes #781 and #782

### LINKED ISSUE #781: [Collecticiel]Appréciation
Bug après avoir fait ceci : 

- Choix de l'appréciation
- Clic sur le bouton bleu Gérer les évaluations
- Construction de l'échelle
- Suppression du critère proposé par défaut
- Ajout de trois critères
- Clic sur Sauvegarder

Après multiples tests, ceci se produit lorsqu’on supprime le critère proposé par défaut (Commentaire)

Message d'erreur : 

Notice: Undefined index: criteriaName in /var/www/demo-v6.innovalangues.net/vendor/claroline/distribution/plugin/collecticiel/Manager/GradingCriteriaManager.php on line 48

Erreur interne du serveur. Veuillez contactez l'administrateur de la plateforme.

### PR DESCRIPTION
**Pull request by ericvincenterv**
bug fixes : 
 [Collecticiel]Appréciation https://github.com/claroline/Distribution/issues/781
[Collecticiel] Appréciation - Gérer les évaluations - Non sauvegarde formulaire de base https://github.com/claroline/Distribution/issues/782

### CODE REVIEW

File: plugin/collecticiel/Resources/views/Dropzone/editCommon.html.twig [hunk 1]
```diff
@@ -153,10 +153,11 @@
                <div class="col-md-12">
                  <div id="dropzone-message-part-appreciation" {% if dropzone.evaluationType != "ratingScale" %} style="display:none" {% endif %}>
                     <hr>
```
Comment (purplefish32) [inline plugin/collecticiel/Resources/views/Dropzone/editCommon.html.twig]: You have some funky indentation going on in this file

Comment (ericvincenterv) [inline plugin/collecticiel/Resources/views/Dropzone/editCommon.html.twig]: OK. Done : https://github.com/claroline/Distribution/pull/826/commits/f5df877a2e3ba283f374b240f3760332c7af4b12

File: plugin/collecticiel/Controller/DropzoneController.php [hunk 2]
```diff
@@ -519,4 +518,88 @@ private function handleFormErrors($form, Dropzone $dropzone)
...
+        $dropzone->setUsername($username);
+
+        // Manage dropzone type and dates
+        if ($manualPlanning == 1) {
+            $dropzone->setManualPlanning($manualPlanning);
+            $dropzone->setManualState($manualState);
+        } else {
+            $dropzone->setManualPlanning($manualPlanning);
+            $formatDate = explode('/', $startAllowDrop_date);
+            $startAllowDrop = $formatDate[2].'-'.$formatDate[1].'-'.$formatDate[0].' '.$startAllowDrop_time.':00';
```
Comment (ericvincenterv) [inline plugin/collecticiel/Controller/DropzoneController.php]: In my action, I have this declaration : 
`        $form = $this->createForm(
            new DropzoneCommonType(), $dropzone,
            array('language' => $platformConfigHandler->getParameter('locale_language'), 'date_format' => 'dd/MM/yyyy')
        );
`

Comment (ptsavdar) [inline plugin/collecticiel/Controller/DropzoneController.php]: Ok then. If I got it right it looks like the date is always in dd/MM/yyyy format. So there are no concerns about another format. But does this also mean that for every language the format is always the same?

Comment (ericvincenterv) [inline plugin/collecticiel/Controller/DropzoneController.php]: The original code is about "DropZoneBundle" (Icap). I don't know if the format is always the same the every language. In think the question was not asked.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ptsavdar** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**